### ISSUE TITLE
Loading Existing Config Package Causing Exception

### ISSUE THREAD

**Issue by davetcoleman** [moveit/moveit_setup_assistant#6]
Loading a previously created moveit_config package for the robonaut causes the application to crash with a malloc error.

**Comment by davetcoleman** [comment]
I have traced the error coming from the srdf package. When a previous SRDF file is initialized using the srdf::Model::initString() function calling the srdf::Model::getDisabledCollisionPairs() is throwing an exception for some reason. This is called from the Setup Assistant from the tools/srdf_writer.cpp file in the initString() function. I am not sure what is wrong within SRDF because when I tried to checkout the source it says the package has been depreciated. Within a Qt application you are not allowed to throw exceptions, so gdb does not work properly in back tracing the error. I can email two different moveit config packages that are causing this error.
